### PR TITLE
Airbnb CSS in Azerbijanian Language

### PR DESCRIPTION
**Pull request by Simuratli**
Translation Airbnb CSS into Azerbaijanian language

### CODE REVIEW

**Comment by Simuratli** [comment]
Hi I am Azerbaijani developer. I translate Airbnb CSS into our language.

File: README.md [hunk 1]
```diff
@@ -310,7 +315,8 @@ If you must use an ID selector in the first place (and you should really try not
   - ![es](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Spain.png) **Spanish**: [ismamz/guia-de-estilo-css](https://github.com/ismamz/guia-de-estilo-css)
   - ![vn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Vietnam.png) **Vietnamese**: [trungk18/css-style-guide](https://github.com/trungk18/css-style-guide)
   - ![vn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Italy.png) **Italian**: [antoniofull/linee-guida-css](https://github.com/antoniofull/linee-guida-css)
-  - ![de](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Germany.png) **German**: [tderflinger/css-styleguide](https://github.com/tderflinger/css-styleguide)
+  - ![de](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Germany.png) **German**: [tderflinger/css-styleguide](https://github.com/tderflinger/css-styleguide);
+  - ![az](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Azerbaijan.png) **Azerbaijanian**: [https://github.com/Simuratli/airbnb-css-az)
```
Comment (sharmilajesupaul) [inline README.md:319]: I think you meant to commit this line and nothing else?

Comment (Simuratli) [reply README.md:319]: İ just add my link into this repo :)

**Comment by ljharb** [comment]
@Simuratli because you made the PR from your master branch - which contains the translation as well - this PR includes the translation too.

What you'll want to do is save your translation changes in another branch, and then force push `master` back to *just* adding the link, and then once this PR is merged, update your master branch to include your translations.

To have avoided this, you could have made the PR from a *different* branch that only differed from this repo by adding the link.

**Pull request closed without merging**